src/Pages/HugeAss/HugeAss.js

```diff
@@ -1,9 +1,105 @@
+                                    target="_blank"
+                                    rel="noopener noreferrer"
+                                >
+                                    subscribing to Patreon.
+                                </a>
+                            </span>
+                        </p>
+                        <div className="beerGogglesListenOnContainer">
+                            <h3 className="listenOnIconAndTitle">
+                                <img
+                                    className="podIcon"
+                                    src={podIcon}
+                                    alt="podcatcher icon"
+                                />
+                                Subscribe On:
+                            </h3>
+                            <img
+                                className="youtubeIcon"
+                                src={youtubeIcon}
+                                alt="youtube triangle icon"
+                            />
+                            <span className="listenOnChannel">
+                                <a
+                                    href="https://www.youtube.com/@HoppedUpGamingEast/featured"
+                                    target="_blank"
+                                    rel="noopener noreferrer"
+                                >
+                                    Youtube
+                                </a>
+                            </span>
+                        </div>
+
+                        <div className="listenOnDiscord">
+                            <img
+                                className="logo"
+                                src={discordIcon}
+                                alt="discord logo"
+                            />
+                            <a
+                                href="https://bit.ly/hoppedupdiscord"
+                                target="_blank"
+                                rel="noopener noreferrer"
+                            >
+                                <p className="listenOnDiscordText">
+                                    Discord Server
+                                </p>
+                            </a>
+                        </div>
+                    </div>
+                    <figure>
+                        <iframe
+                            width="450"
+                            height="315"
+                            title="YouTubeEmbed HUGE ASS Playlist"
+                            src="https://www.youtube.com/embed/videoseries?list=PLXnsyGNMKFBgo8lYTARS244KlTaO1wTXb"
+                        ></iframe>
+                        <figcaption className="hugeAssfigCapt">
+                            Season One Playlist
+                        </figcaption>
+                    </figure>
+                </div>
+                <Patreon
+                    patreonTitle="Get Season Two Now on Patreon"
```

Comment:
```suggestion
                    patreonTitle="Season Two Early Access on Patreon"
```
The whole season isn't out yet so I don't want to overpomise lol